src/Symfony/Component/Validator/Tests/Resources/TranslationFilesTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Resources;
+
+class TranslationFilesTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        libxml_use_internal_errors(true);
+        libxml_clear_errors();
```

Comment:
shouldn't you do this in the foreach loop below ?
I'd suggest also to move `libxml_use_internal_errors(true);` in the test case